manager/controllers/baseline_create.go

```diff
@@ -16,16 +16,19 @@ import (
 )
 
 type CreateBaselineRequest struct {
-	// Baseline name (optional).
+	// Baseline name.
 	Name *string `json:"name"`
 	// Inventory IDs list of systems to associate with this baseline (optional).
 	InventoryIDs []string `json:"inventory_ids"`
 	// Baseline config to filter applicable advisories and package updates for the associated systems (optional).
 	Config *BaselineConfig `json:"config"`
+	// Description of the baseline (optional).
+	Description string `json:"description"`
 }
 
 type CreateBaselineResponse struct {
-	BaselineID int `json:"baseline_id" example:"1"` // Updated baseline unique ID, it can not be changed
+	BaselineID  int    `json:"baseline_id" example:"1"` // Updated baseline unique ID, it can not be changed
+	Description string `json:"description,omitempty"`
```

Comment:
Do not add description to the response. UI already have this data, it put them into request creating the baseline.

Reply:
Ok